src/utils/isEqual.ts

```diff
@@ -0,0 +1,35 @@
+export function isEqual<T>(obj1: T, obj2: T): boolean {
+  if (obj1 === obj2) return true;
+
```

Comment:
```
NaN === NaN 
```
There's probably a few more cases.

i think `lodash.isequal` is already installed :)